File: lists/alternate.py


def alternate(l1, l2):
    result = []
    x = 0
    while x < len(l1):
        result.append(l1[x])
        result.append(l2[x])
        x += 1
    return result 

temp1 = ['a', 'b', 'c']
temp2 = [1, 2, 3]

print(alternate(temp1, temp2))

File: lists/alternate.js


const alternate = (arr1, arr2) => {
    let result = [];
    for (let x = 0; x < arr1.length; x++) {
        result.push(arr1[x]);
        result.push(arr2[x]);
    }

    return result;
}

let a = ['a', 'b', 'c'];
let b = [1, 2, 3]

console.log(alternate(a, b));